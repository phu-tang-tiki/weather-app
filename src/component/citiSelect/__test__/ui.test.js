import CitiSelect from '../index';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

describe('citi select effect', () => {
  const middlewares = [];
  const mockStore = configureStore(middlewares);
  let store;
  store = mockStore({});
  describe('Citi select index', () => {
    it('renders correctly', () => {
      const { asFragment } = render(
        <Provider store={store}>
          <CitiSelect />
        </Provider>
      );

      expect(asFragment()).toMatchSnapshot();
    });
  });
});
