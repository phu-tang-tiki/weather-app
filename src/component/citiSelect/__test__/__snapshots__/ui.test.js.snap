// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`citi select effect Citi select index renders correctly 1`] = `
<DocumentFragment>
  <div
    class="ant-select ant-select-single ant-select-allow-clear ant-select-show-search"
    style="width: 200px;"
  >
    <div
      class="ant-select-selector"
    >
      <span
        class="ant-select-selection-search"
      >
        <input
          aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
          aria-autocomplete="list"
          aria-controls="rc_select_TEST_OR_SSR_list"
          aria-haspopup="listbox"
          aria-owns="rc_select_TEST_OR_SSR_list"
          autocomplete="off"
          class="ant-select-selection-search-input"
          id="rc_select_TEST_OR_SSR"
          role="combobox"
          type="search"
          value=""
        />
      </span>
      <span
        class="ant-select-selection-placeholder"
      >
        Search the city
      </span>
    </div>
    <span
      aria-hidden="true"
      class="ant-select-clear"
      style="user-select: none;"
      unselectable="on"
    >
      <span
        aria-label="close-circle"
        class="anticon anticon-close-circle"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="close-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
          />
        </svg>
      </span>
    </span>
  </div>
</DocumentFragment>
`;
